lib/network.go

```diff
@@ -201,66 +201,74 @@ type DeSoMessage interface {
 type TxnType uint8
 
 const (
-	TxnTypeUnset                        TxnType = 0
-	TxnTypeBlockReward                  TxnType = 1
-	TxnTypeBasicTransfer                TxnType = 2
-	TxnTypeBitcoinExchange              TxnType = 3
-	TxnTypePrivateMessage               TxnType = 4
-	TxnTypeSubmitPost                   TxnType = 5
-	TxnTypeUpdateProfile                TxnType = 6
-	TxnTypeUpdateBitcoinUSDExchangeRate TxnType = 8
-	TxnTypeFollow                       TxnType = 9
-	TxnTypeLike                         TxnType = 10
-	TxnTypeCreatorCoin                  TxnType = 11
-	TxnTypeSwapIdentity                 TxnType = 12
-	TxnTypeUpdateGlobalParams           TxnType = 13
-	TxnTypeCreatorCoinTransfer          TxnType = 14
-	TxnTypeCreateNFT                    TxnType = 15
-	TxnTypeUpdateNFT                    TxnType = 16
-	TxnTypeAcceptNFTBid                 TxnType = 17
-	TxnTypeNFTBid                       TxnType = 18
-	TxnTypeNFTTransfer                  TxnType = 19
-	TxnTypeAcceptNFTTransfer            TxnType = 20
-	TxnTypeBurnNFT                      TxnType = 21
-	TxnTypeAuthorizeDerivedKey          TxnType = 22
-	TxnTypeMessagingGroup               TxnType = 23
-	TxnTypeDAOCoin                      TxnType = 24
-	TxnTypeDAOCoinTransfer              TxnType = 25
-	TxnTypeDAOCoinLimitOrder            TxnType = 26
-
-	// NEXT_ID = 27
+	TxnTypeUnset                         TxnType = 0
+	TxnTypeBlockReward                   TxnType = 1
+	TxnTypeBasicTransfer                 TxnType = 2
+	TxnTypeBitcoinExchange               TxnType = 3
+	TxnTypePrivateMessage                TxnType = 4
+	TxnTypeSubmitPost                    TxnType = 5
+	TxnTypeUpdateProfile                 TxnType = 6
+	TxnTypeUpdateBitcoinUSDExchangeRate  TxnType = 8
+	TxnTypeFollow                        TxnType = 9
+	TxnTypeLike                          TxnType = 10
+	TxnTypeCreatorCoin                   TxnType = 11
+	TxnTypeSwapIdentity                  TxnType = 12
+	TxnTypeUpdateGlobalParams            TxnType = 13
+	TxnTypeCreatorCoinTransfer           TxnType = 14
+	TxnTypeCreateNFT                     TxnType = 15
+	TxnTypeUpdateNFT                     TxnType = 16
+	TxnTypeAcceptNFTBid                  TxnType = 17
+	TxnTypeNFTBid                        TxnType = 18
+	TxnTypeNFTTransfer                   TxnType = 19
+	TxnTypeAcceptNFTTransfer             TxnType = 20
+	TxnTypeBurnNFT                       TxnType = 21
+	TxnTypeAuthorizeDerivedKey           TxnType = 22
+	TxnTypeAccessGroupCreate             TxnType = 23
+	TxnTypeDAOCoin                       TxnType = 24
+	TxnTypeDAOCoinTransfer               TxnType = 25
+	TxnTypeDAOCoinLimitOrder             TxnType = 26
+	TxnTypeAccessGroupAddMember          TxnType = 27
+	TxnTypeAccessGroupRemoveMember       TxnType = 28
+	TxnTypeAccessGroupSetMemberAttribute TxnType = 29
+	TxnTypeAccessGroupSetGroupAttribute  TxnType = 30
```

Comment:
wait we're adding new transaction types for each of these? It would be great if we could minimize the number of new txn types. Also, isn't adding member to access groups already a part of an existing txn? some clarity on why we're adding new txn types would be helpful